examples/interactionsDemo.html

```diff
@@ -214,12 +198,12 @@ <h2>Manual Tester</h2>
                 let canViewInteraction = false;
                 let canViewWrapUpCodes = false;
 
-                client.loginImplicitGrant(clientId, redirectUrl, { state: ('pcEnvironment=' + pcEnvironment) })
-                    .then(function () {
+                authenticate(client, pcEnvironment)
+                    .then(() => {
                         authenticated = true;
-                        return new platformClient.UsersApi().getUsersMe({ 'expand': ['authorization'] });
+                        new platformClient.UsersApi().getUsersMe({ 'expand': ['authorization'] })
```

Comment:
Was this an intentional removal of the return statement?  Don't we want to wait on the resolution of this promise for the next then block?

Reply:
Nice catch 👍 Definitely need to return here. Just looked at this a bit and funny enough, since we only do the boolean flag checking, we fail silently here and so I wasn't seeing errors. `profileData` is undefined on 206 and we try to read `profileData.authorization.permissions` on 210 which throws a type error, but since we set both `authenticated` and `userDataAcquired` to true. The catch block just ignores the error and hides the loader as if nothing happened. Anyway, will get this fixed this up 😄

Reply:
Doh, guess we need an else with a generic message, huh?  Thanks for looking into it!